CMakeLists.txt

```diff
@@ -130,6 +130,10 @@ if(NOT BUILD_DYNAMIC_LIBRARIES AND NOT BUILD_STATIC_LIBRARIES)
         "are both set to FALSE. No compilation is possible!")
 endif()
 
+# ADOL-C libraries and tests won't be built unless this is set.
```

Comment:
Please add a comment noting that the ADOLC libraries are always built shared rather than static (assuming that's correct).

Reply:
@sherm1, I have added the line below (134).